Packs/Censys/Integrations/CensysV2/CensysV2_test.py

```diff
@@ -0,0 +1,167 @@
+    Then:
+        Validate the output compared to the mock output
+    """
+    args = {
+        'index': 'certificates',
+        'query': "parsed.issuer.common_name: \"Let's Encrypt\"",
+        'fields': ['parsed.fingerprint_sha1', 'validation.apple.valid']
+    }
+
+    mock_response = util_load_json('test_data/search_certs_response.json')
+    mocker.patch.object(client, '_http_request', return_value=mock_response)
+    mocker.patch.object(demisto, 'args', return_value={'fields': 'parsed.fingerprint_sha1'})
+    response = censys_search_command(client, args)
+    assert client._http_request.call_args.kwargs['json_data']['fields'] == ['parsed.fingerprint_sha256',
+                                                                            'parsed.subject_dn', 'parsed.issuer_dn',
+                                                                            'parsed.issuer.organization',
+                                                                            'parsed.validity.start',
+                                                                            'parsed.validity.end', 'parsed.names',
+                                                                            'parsed.fingerprint_sha1',
+                                                                            'validation.apple.valid']
+    assert "### Search results for query \"parsed.issuer.common_name: \"Let's Encrypt\"" in response.readable_output
+    assert response.outputs == SEARCH_CERTS_OUTPUTS
+
+
+def test_censys_view_host(requests_mock, client):
+    """
+    Given:
+        Command arguments: query ip = 8.8.8.8
+    When:
+        Running cen_view_command
+    Then:
+        Validate the output compared to the mock output
+    """
+    args = {
+        'index': 'ipv4',
+        'query': "8.8.8.8"
+    }
+    mock_response = util_load_json('test_data/view_host_response.json')
+    requests_mock.get('https://search.censys.io/api/v2/hosts/8.8.8.8', json=mock_response)
+    response = censys_view_command(client, args)
+    assert '### Information for IP 8.8.8.8' in response.readable_output
+    assert response.outputs == mock_response.get('result')
+
+
+def test_censys_view_host_invalid(requests_mock, mocker):
+    """
+    Given:
+        Command arguments: query ip = test
+    When:
+        Running cen_view_command
+    Then:
+        Validate error message returns.
+    """
+    args = {
+        'index': 'ipv4',
+        'query': "test"
+    }
+    mock_response = {
+        "code": 422,
+        "status": "Unprocessable Entity",
+        "error": "ip: value is not a valid IPv4 or IPv6 address"
+    }
+    requests_mock.get('https://search.censys.io/api/v2/hosts/test', json=mock_response, status_code=422)
+    return_error_mock = mocker.patch.object(CensysV2, 'return_error')
+    mocker.patch.object(demisto, 'error')
+    mocker.patch.object(demisto, 'args', return_value=args)
+    mocker.patch.object(demisto, 'command', return_value='cen-view')
+    main()
```

Comment:
are we calling `main` in order to see what `return_error` is called with?
if so, it would probably be better to use `pytest.raises` and call `censys_view_command` here instead